purchase_shop/models/purchase_order.py

```diff
@@ -58,13 +58,19 @@ def onchange_shop_id(self):
     @api.multi
     def write(self, vals):
         res = super().write(vals)
-        if "order_line" in vals or "shop_id" in vals or "purchased_by_id" in vals:
+        if (
+            "order_line" in vals
+            or "shop_id" in vals
+            or "purchased_by_id" in vals
+            or "evaluated_by_id" in vals
+        ):
             for order in self:
                 products = order.order_line.mapped("product_id")
                 products.write(
                     {
                         "shop_id": order.shop_id.id,
                         "purchased_by_id": order.purchased_by_id.id,
+                        "evaluated_by_id": order.employee_id.id,
```

Comment:
As per current design, I think purchase order creation will not assign the these values in products. I believe we should adjust the `create()` method as well.

Reply:
We can pass tha value with current design in the creation of purchase_order.  Can you try it in test server?

Reply:
I tested in my dev env. As I mentioned values are not assigned in product. You can see this order https://dev3.quartile.co/web?debug=1#id=11&view_type=form&model=purchase.order&menu_id=213&action=313.

Reply:
I had to install the `purchase_ext_sst` module for it to work in your environment. I think the cause was that the `employee_id` was not in the purchase.order.
I updated that point.  Can you try again?

Reply:
Now I know the workflow. It seems like customer create the purchase order first in tree view and then adding order line after that. Thats why current design is covered for this workflow. But what if they change to kanban view and create purchase order? I am not pretty sure customer will do like this or not but as a design, it is not covered for possible purchase order creation.

Reply:
`<attribute name="context">{"default_evaluated_by_id": parent.employee_id}</attribute>` If this feature is covered, there is no problem. As noted in the opening message, it was given as a default value in context.